### PR TITLE
patch: Account synchronization improvements (python paged search) #1664

### PR DESCRIPTION
**Pull request by vokac**
patch: Account synchronization improvements (python paged search) #1664

### CODE REVIEW

**Comment by bari12** [comment]
Thanks for the patch 😄 

The styleguide test failed with:

```
tools/probes/common/check_voms:15:1: F401 're' imported but unused
tools/probes/common/check_voms:35:15: E201 whitespace after '['
tools/probes/common/check_voms:35:39: E202 whitespace before ']'
tools/probes/common/check_voms:38:5: E265 block comment should start with '# '
tools/probes/common/check_voms:39:5: E265 block comment should start with '# '
tools/probes/common/check_voms:40:5: E265 block comment should start with '# '
tools/probes/common/check_voms:46:1: E265 block comment should start with '# '
tools/probes/common/check_voms:48:65: E202 whitespace before ']'
tools/probes/common/check_voms:48:67: E261 at least two spaces before inline comment
tools/probes/common/check_voms:74:25: E201 whitespace after '['
tools/probes/common/check_voms:74:69: E202 whitespace before ']'
tools/probes/common/check_voms:74:75: E201 whitespace after '['
tools/probes/common/check_voms:74:83: E202 whitespace before ']'
tools/probes/common/check_voms:88:17: E265 block comment should start with '# '
tools/probes/common/check_voms:91:26: E201 whitespace after '['
tools/probes/common/check_voms:91:53: E202 whitespace before ']'
tools/probes/common/check_voms:92:37: E701 multiple statements on one line (colon)
tools/probes/common/check_voms:96:57: E701 multiple statements on one line (colon)
tools/probes/common/check_voms:104:13: E265 block comment should start with '# '
```

**Comment by bari12** [comment]
I think there are two new flake8 errors 😄 
```
tools/probes/common/check_voms:15:1: F401 're' imported but unused
tools/probes/common/check_voms:46:1: E265 block comment should start with '# '
```

File: tools/probes/common/check_voms [hunk 1]
```diff
@@ -11,13 +11,13 @@
...
 # - David Cameron, <david.cameron@cern.ch>, 2015
 
-import commands
 import os
 import re
 import sys
 import time
 import types
 
+import ldap
```
Comment (mlassnig) [inline tools/probes/common/check_voms:19]: this has to be added to the requirements.txt?

Comment (bari12) [reply tools/probes/common/check_voms:19]: Since it is only a probe, I wouldn't add it.

**Comment by vokac** [comment]
Hopefully last attempt to fix all remaining coding style issues.
Can somebody test full functionality of this script, because I don't have access rucio DB and also my credentials are not sufficient to list details from VOMS server. Basically I tested only my new function that replaces ldapsearch with python paged ldap query and I were not able to really run whole script with minor code updates also in the main function.

**Comment by bari12** [comment]
Thanks a lot. Yes I am sure @cserf and @dchristidis plan some functional testing.

**Comment by cserf** [comment]
Test looks OK. BTW, you forgot to put your name in the list of authors

**Review by cserf — APPROVED** [review]

**Pull request merged**